### DISCUSSION TITLE
Data protection  - is that an issue?

### DISCUSSION THREAD

**Discussion by KennyDenvers** [brianpetro/obsidian-smart-connections#340] [Q&A] [answered]
Hi, I have been reluctant using any of the AI / ChatGPT plugins for Obsidian so far due to concerns about data protection. Therefore I have not yet tried out any of those plugins, including this one. 
My technical understanding is somewhat limited therefore I would very much appreciate some input from those of you that will have more expertise in this:
My question / concern goes: Will using one of these AI plugins (including this one) make that all of my notes are sent through the internet to OpenAI for processing - or how does this actually work?
Not that all of my notes are super-confidential but there some personal enough that I would not want to share them with the whole world.
Any comments and feedback much appreciated!

**Comment by brianpetro** [comment, marked as answer]
Short answer: yes, your notes are sent to OpenAI servers for processing.

Smart Connections has "exclusion" options in the settings to prevent pre-specified notes and folders from being included in that processing. 

As long as you set the exclusion settings *prior to* setting your API Key (and subsequently clicking the "Create Embeddings" button) then the excluded notes will not be sent to OpenAI.

There is quite a bit of interest in local models which would prevent sending notes over the internet for processing. This is an area that's very much in active development (outside of Smart Connections) and something I'm keeping a close eye on for future integration into this plugin.

Hope that helps,
🌴 Brian

**Reply by KennyDenvers** [reply]
Thank you for your prompt response! Yes, that helps!
Kenny
